docs/admin/install.md

```diff
@@ -91,12 +76,12 @@ The plugin behavior can be controlled with the following list of settings:
 
 ## Integrations Configuration
 
-The `nautobot-ssot` package includes multiple integrations. Each requires extra dependencies defined in `pyproject.toml`. See the [installation guide](#installation-guide) for instructions on installing these dependencies.
+The `nautobot-ssot` package includes multiple integrations. Each requires extra dependencies defined in `pyproject.toml`. See the [install guide](#install-guide) for instructions on installing these dependencies.
```

Comment:
```suggestion
The `nautobot-ssot` package includes multiple integrations. Each requires extra dependencies defined in `pyproject.toml`.
```

Reply:
No need to distract by sending them to the start again, each integration setup page reinforces the extras that need to be installed.